specs/interop/predeploys.md

```diff
@@ -198,7 +198,7 @@ function relayMessage(uint256 _destination, uint256 _nonce, address _sender, add
     require(msg.sender == address(CROSS_L2_INBOX));
     require(_destination == block.chainid);
     require(CROSS_L2_INBOX.origin() == address(this));
-    require(_target != address(this));
+    require(_target != address(CROSS_L2_INBOX));
```

Comment:
The `msg.sender != tx.origin` check achieves this, we do want to make sure that we cannot reenter this function from within itself altho `require(msg.sender == address(CROSS_L2_INBOX)` ensures this so we could probably just delete this check

Reply:
Now that the code exists, we could turn this sample implementation into a list of invariants

Reply:
done. just deleted the line!